plugins/woocommerce/src/Internal/Fulfillments/FulfillmentsRenderer.php

```diff
@@ -117,6 +118,7 @@ private function render_fulfillment_status_column_row_data( WC_Order $order, arr
 				<path d='M11.8333 2.83301L9.33329 0.333008L2.24996 7.41634L1.41663 10.7497L4.74996 9.91634L11.8333 2.83301ZM5.99996 12.4163H0.166626V13.6663H5.99996V12.4163Z' fill='#3858E9'/>
```

Comment:
The inline `fill` in the `path` element will take preference over any style from a stylesheet.

Reply:
Thanks for pointing that out! I'll remove the `fill` property so it can use the one from CSS.

Reply:
Or, I'll do the change on the next PR if it's ok.

Reply:
Sure!